platform-controller/src/test/java/org/hobbit/controller/docker/ContainerManagerBasedTest.java

```diff
@@ -20,6 +20,7 @@
 import java.util.List;
 
 import org.hobbit.controller.DockerBasedTest;
+import org.hobbit.controller.containers.docker.ContainerManagerImpl;
```

Comment:
When the tested class moves into a different package, the test classes should move, too.